repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/StepStatus.java

```diff
@@ -3,6 +3,10 @@
 import fr.inria.spirals.repairnator.process.step.AbstractStep;
 import fr.inria.spirals.repairnator.states.PipelineState;
 
+/**
+ * This class defines how an AbstractStep has been executed.
```

Comment:
Is it not better to move `StepStatus` to the same package than `AbstractStep` is?

Reply:
I can check that